packages/bigcommerce-payments-integration/src/BigCommercePaymentsPaypal/BigCommercePaymentsPaymentMethod.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FunctionComponent, useEffect } from 'react';
+
+import {
+    AccountInstrumentFieldset,
+    StoreInstrumentFieldset,
+} from '@bigcommerce/checkout/instrument-utils';
+import { TranslatedHtml } from '@bigcommerce/checkout/locale';
+import {
+    PaymentMethodProps,
+    PaymentMethodResolveId,
+    toResolvableComponent,
+} from '@bigcommerce/checkout/payment-integration-api';
+import { LoadingOverlay } from '@bigcommerce/checkout/ui';
+
+import BigCommercePaymentsPaymentMethodComponent from '../components/BigCommercePaymentsPaymentMethodComponent';
+import useBigCommercePaymentsInstrument from '../hooks/useBigCommercePaymentsInstruments';
+
+const BigCommercePaymentsPaymentMethod: FunctionComponent<PaymentMethodProps> = (props) => {
```

Comment:
```suggestion
const BigCommercePaymentsPayPalPaymentMethod: FunctionComponent<PaymentMethodProps> = (props) => {
```

And please, update file name

Reply:
fixed